src/main/scala/viper/gobra/translator/transformers/RefuteTransformer.scala

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2021 ETH Zurich.
+
+package viper.gobra.translator.transformers
+
+import viper.gobra.backend.BackendVerifier
+import viper.silver.plugin.SilverPlugin
+import viper.silver.plugin.standard.refute.RefutePlugin
+import viper.silver.verifier.AbstractError
+import viper.silver.{ast => vpr}
+
+class RefuteTransformer extends ViperTransformer {
+  override def transform(task: BackendVerifier.Task): Either[Seq[AbstractError], BackendVerifier.Task] = {
+    for {
+      transformedProg <- executeTerminationPlugin(task)
+    } yield transformedProg
```

Comment:
This thing is the same as 
```suggestion
    executeTerminationPlugin(task)
```
which makes me wonder if we need to have the logic on a separate function. Maybe we should just inline `executeTerminationPlugin` (a.k.a. `executeRefutePlugin`) here.

Reply:
-> [ed6fbf4](https://github.com/viperproject/gobra/pull/776/commits/ed6fbf44f1e9990a4c9331f60eb3ea38f1fc3c44)